unstructured/documents/elements.py

```diff
@@ -265,6 +269,70 @@ def get_last_modified(self) -> Optional[datetime.datetime]:
         return dt
 
 
+class ConsolidationStrategy(enum.Enum):
+    """Methods by which a metadata field can be consolidated across a collection of elements.
+
+    These are assigned to `ElementMetadata` field-names immediately below. Metadata consolidation is
+    part of the chunking process and may arise elsewhere as well.
+    """
+
+    DROP = "drop"
+    """Do not include this field in the consolidated metadata object."""
```

Comment:
These enum-member docstrings show up in VisualStudio and other IDE "Intellisense" completion, so seem a better choice than comments for documenting these.